crates/proof-of-sql/src/proof_primitive/dory/setup.rs

```diff
@@ -73,6 +73,18 @@ impl<'a> ProverSetup<'a> {
     ) {
         self.blitzar_handle.msm(res, element_num_bytes, scalars)
     }
+
+    #[cfg(feature = "blitzar")]
+    #[tracing::instrument(name = "ProverSetup::blitzar_packed_msm", level = "debug", skip_all)]
+    pub(super) fn blitzar_packed_msm(
+        &self,
+        res: &mut [blitzar::compute::ElementP2<ark_bls12_381::g1::Config>],
+        output_bit_table: &[u32],
+        scalars: &[u8],
+    ) {
+        self.blitzar_handle
+            .packed_msm(res, output_bit_table, scalars)
```

Comment:
Can you bump the version of `blitzar` from `3.0.2` to `3.1.0` in the `Cargo.toml` file to ensure that `packed_msm` is included?